src/qt/forms/intro.ui

```diff
@@ -223,7 +223,7 @@
    <item>
     <widget class="QLabel" name="lblExplanation2">
      <property name="text">
-      <string>This initial synchronisation is very demanding, and may expose hardware problems with your computer that had previously gone unnoticed. Each time you run %1, it will continue downloading where it left off.</string>
```

Comment:
"Synchronisation" is just British spelling of American "synchronization", don't see a need for this to be changed.

Reply:
Yes, I saw in other places the American spelling was used so I thought it should be changed for consistency.